.github/workflows/stage_pr.yaml

```diff
@@ -2,8 +2,7 @@ name: "Deploy Pull Request"
 
 on: 
   pull_request:
-    branches: main
-    types: [labeled, opened, synchronize]
+      types: [opened, synchronize]
```

Comment:
What is synchronize?

Reply:
Synchronize is on branch update, so when new code is pushed to the branch.